net-libs/libcloudproviders/libcloudproviders-0.3.1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+VALA_USE_DEPEND=vapigen
+
+inherit gnome.org meson vala
+
+DESCRIPTION="DBus API that allows cloud storage sync clients to expose their services"
+HOMEPAGE="https://gitlab.gnome.org/World/libcloudproviders"
+
+LICENSE="LGPL-3"
+SLOT="0"
+IUSE="gtk-doc +introspection vala"
+
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~riscv ~sparc ~x86 ~amd64-linux ~x86-linux"
+
+DEPEND="
+	>=dev-libs/glib-2.51.2:2
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	>=dev-util/meson-0.47.0
+	dev-util/glib-utils
+	gtk-doc? ( >=dev-util/gtk-doc-1.10 )
+	vala? ( $(vala_depend) )
+"
+
+src_prepare() {
+	default
+	use vala && vala_setup && export VALA_API_GEN="${VAPIGEN}"
+}
+
+src_configure() {
+	local emesonargs=(
+		$(meson_use gtk-doc enable-gtk-doc)
+		$(meson_use introspection)
+		-Dinstalled-tests=false
```

Comment:
Let's keep these in the same order as `meson_options.txt`. `enable-gtk-doc` / `installed-tests` / `introspection` / `vapigen`.